File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.py
def solution(a):
    answer = []
    while(a > 0):
        answer.append(a % 3)
        a = a // 3
    t = 0
    index = 0
    for i in range(len(answer)-1, -1, -1):
        t += answer[i] * (3**index)
        index += 1
    return t

File: 프로그래머스/1/68935. 3진법 뒤집기/3진법 뒤집기.swift
import Foundation

func solution(_ n:Int) -> Int {
    var answer : Int = 0
    var N : Int = n
    var array : [Int] = []
    while ( N > 0 ) {
        array.append(N % 3)
        N /= 3
    }
    array.reverse()
    for index in 0..<array.count {
        answer += (array[index]) * Int((pow(3.0, Double(index))))
    }
    return answer
}